### DISCUSSION TITLE
Compared to MathDotNet.Numerics

### DISCUSSION THREAD

**Discussion by Arithmomaniac** [USArmy-Corps-of-Engineers-RMC/Numerics#113] [General]
I was considering this library, but wanted to understand its rationale given that MathDotNet.Numerics is already pretty robust (if old):
1. Why did USACE-RMC develop this library when MathDotNet exists?
2. Is this provided "as-is", or do you intend to provide community support, accept PRs, etc?

**Comment by HadenSmith** [comment]
Hi,
I apologize for the delayed response; I just now saw your comment.

1. This library was developed over several years to support the full RMC software suite, which includes specialized risk analysis and statistical methods. While there is some overlap with MathDotNet in certain classes and methods, many routines in this library are unique and more specialized. We wanted to make these tools available to the public to broaden access to these specialized capabilities.


2. The library is provided primarily "as-is." However, we plan to create a wiki as time allows, and we welcome pull requests. Please note that these contributions will also be reviewed as our time permits.
